### PR TITLE
create extension

### PR DESCRIPTION
**Pull request by khanh-kyo**

### CODE REVIEW

**Comment by tolgap** [comment]
@khanh-kyo thanks for considering contributing to this project ❤️ 

I don't think extensions are within the scope of this example app. They have nothing to do with NestJS itself. It's up to each developer on how they want to consume the `@nestjs-shopify/*` packages. This repo is only for "kind of" showing how to demo a NestJS app.

And Shopify apps aren't really demo-able without a frontend. So I chose to add a NextJS app as well.

In other words: I would encourage you to setup your own repo where you demo your own structure. That way we can keep this repo minimal to just demo our NestJS backend 👍.
